src/widgets/dialogs/LastRunCrashDialog.cpp

```diff
@@ -1,93 +1,162 @@
-#include "LastRunCrashDialog.hpp"
+#include "widgets/dialogs/LastRunCrashDialog.hpp"
 
-#include "singletons/Updates.hpp"
+#include "common/Args.hpp"
+#include "common/Literals.hpp"
+#include "common/Modes.hpp"
+#include "singletons/Paths.hpp"
 #include "util/LayoutCreator.hpp"
-#include "util/PostToThread.hpp"
 
+#include <QDesktopServices>
 #include <QDialogButtonBox>
+#include <QDir>
 #include <QLabel>
+#include <QMessageBox>
+#include <QProcess>
 #include <QPushButton>
+#include <QRandomGenerator>
+#include <QStringBuilder>
 #include <QVBoxLayout>
 
+namespace {
+
+using namespace chatterino::literals;
+
+const std::vector<QString> MESSAGES = {
+    u"Oops..."_s,
+    u"NotLikeThis"_s,
+    u"NOOOOOO"_s,
+    u"I'm sorry"_s,
+    u"We're sorry"_s,
+    u"My bad"_s,
+    u"FailFish"_s,
+    u"O_o"_s,
+    uR"("%/§*'"$)%=})"_s,
+    u"Sorry :("_s,
+    u"I blame cosmic rays"_s,
+    u"I blame TMI"_s,
+    u"I blame Helix"_s,
+    // "Wtf is Utf16?" (but with swapped endian)
+    u"圀琀昀\u2000椀猀\u2000唀琀昀㄀㘀㼀"_s,
+    u"Oopsie woopsie"_s,
+};
+
+QString randomMessage()
+{
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
+    using Ty = quint32;
+#else
+    using Ty = quint64;
+#endif
+    return MESSAGES[static_cast<size_t>(
```

Comment:
Just use modulo here, no need for a qt version check

Reply:
This isn't uniform now.